Rules/UseConsistentIndentation.cs

```diff
@@ -284,6 +287,34 @@ private static bool PipelineIsFollowedByNewlineOrLineContinuation(Token[] tokens
             return false;
         }
 
+        private static bool PreviousLineEndedWithPipe(Token[] tokens, int tokenIndex, Token token)
+        {
+            if (tokenIndex < 2 || token.Extent.StartLineNumber == 1)
+            {
+                return false;
+            }
+
+            int searchIndex = tokenIndex - 2;
+            int searchLine = token.Extent.StartLineNumber;
```

Comment:
This line is always reassigned, so is it possible to simply declare it without initialization here?

Reply:
```suggestion
            int searchLine;
```